vm/actor/src/builtin/cron/state.rs

```diff
@@ -2,62 +2,21 @@
 // SPDX-License-Identifier: Apache-2.0, MIT
 
 use address::Address;
-use encoding::Cbor;
-use serde::{Deserialize, Deserializer, Serialize, Serializer};
+use encoding::{tuple::*, Cbor};
+use serde::{Deserialize, Serialize};
 use vm::MethodNum;
 
 /// Cron actor state which holds entries to call during epoch tick
-#[derive(Default)]
+#[derive(Default, Serialize, Deserialize)]
```

Comment:
same here, not correct impl (use `#[serde(transparent)]` and you get what you want